src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Scanner;
+
 public class Duke {
+    static int limitTask = 100;
+    static Task[] tasks = new Task[limitTask];
+    static int taskCount = 0;
+
+
+
+    public static void listTask(){
+        int idxCount = 1;
+        printHorizontalLine();
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0 ; i < taskCount; i++){
+            System.out.println(idxCount + "." + tasks[i].toString());
+            idxCount++;
+        }
+        printHorizontalLine();
+    }
+    public static void addTask(String input, String taskType){
+        if(taskCount == limitTask ) {
+            printHorizontalLine();
+            System.out.println("Too much tasks");
+            printHorizontalLine();
+            return;
+        }
+
+        String task;
+        int firstSpaceAfterTaskType,firstSlashSeparator,secondSlashSeparator;
+
+        switch (taskType){
+            case "todo":
```

Comment:
Case statements should not be indented, you can change your settings in VS Code to reflect that so you don't have to manually delete in the future :)